wakuv2/waku.go

```diff
@@ -57,9 +57,11 @@ import (
 	"github.com/libp2p/go-libp2p/core/metrics"
 
 	"github.com/waku-org/go-waku/waku/v2/dnsdisc"
+	"github.com/waku-org/go-waku/waku/v2/peermanager"
 	wps "github.com/waku-org/go-waku/waku/v2/peerstore"
 	"github.com/waku-org/go-waku/waku/v2/protocol"
 	"github.com/waku-org/go-waku/waku/v2/protocol/filter"
+	"github.com/waku-org/go-waku/waku/v2/protocol/legacy_store"
```

Comment:
Do we need legacy_store, if we are using storev3 for history queries?

Reply:
Sadly, yes, because a test unit requires running its own store node, and this feature was not implemented in storev3 for go-waku